cfme/tests/configure/test_landing_page.py

```diff
@@ -0,0 +1,140 @@
+    'Compute / Containers / Container Images',
+    'Compute / Containers / Container Templates',
+    'Compute / Containers / Topology',
+    'Configuration / Management',
+    'Networks / Providers',
+    'Networks / Networks',
+    'Networks / Subnets',
+    'Networks / Network Routers',
+    'Networks / Security Groups',
+    'Networks / Floating IPs',
+    'Networks / Network Ports',
+    'Networks / Load Balancers',
+    'Networks / Topology',
+    'Middleware / Providers',
+    'Middleware / Domains',
+    'Middleware / Servers',
+    'Middleware / Deployments',
+    'Middleware / Datasources',
+    'Middleware / Messagings',
+    'Middleware / Topology',
+    'Datawarehouse / Providers',
+    'Storage / Block Storage / Managers',
+    'Storage / Block Storage / Volumes',
+    'Storage / Block Storage / Volume Snapshots',
+    'Storage / Block Storage / Volume Backups',
+    'Storage / Object Storage / Managers',
+    'Storage / Object Storage / Object Store Containers',
+    'Storage / Object Storage / Object Store Objects',
+    'Control / Explorer',
+    'Control / Simulation',
+    'Control / Import / Export',
+    'Control / Log',
+    'Automation / Ansible / Playbooks',
+    'Automation / Ansible / Repositories',
+    'Automation / Ansible / Credentials',
+    'Automation / Ansible Tower / Explorer',
+    'Automation / Ansible Tower / Jobs',
+    'Automation / Automate / Explorer',
+    'Automation / Automate / Simulation',
+    'Automation / Automate / Customization',
+    'Automation / Automate / Import / Export',
+    'Automation / Automate / Log',
+    'Automation / Automate / Requests',
+    'Optimize / Utilization',
+    'Optimize / Planning',
+    'Optimize / Bottlenecks',
+    'Monitor / Alerts / Overview',
+    'Monitor / Alerts / All Alerts',
+    'Monitor / Alerts / Most Recent Alerts',
+    'Settings / Tasks',
+    'Red Hat Access Insights']
+
+
+def set_landing_page(value, appliance):
+    # page_list contains the list of pages which show some error or alerts after login.
+    page_list = []
+    view = navigate_to(Visual, 'All')
+    if (not any(substring in value for substring in page_list) and
+            view.visualstartpage.show_at_login.fill(value)):
+        view.save.click()
+    # This block will redirect to My setting page and update the startpage value to next parameter.
+    # except NoSuchElementException:
+    #     browser.start(url_key="{}/configuration/index".format(appliance.server.address))
+    #     view.visualstartpage.show_at_login.fill(value)
+    #     view.save.click()
+    login_page = navigate_to(appliance.server, 'LoginScreen')
+    login_page.login_admin()
+    logged_in_page = navigate_to(appliance.server, 'LoggedIn')
+    return logged_in_page.is_displayed
+
+
+def set_to_default(page):
+    view = navigate_to(Visual, 'All')
+    view.visualstartpage.show_at_login.fill(page)
+    view.save.click()
+
+
+@pytest.mark.parametrize('start_page', LANDING_PAGES, scope="module")
```

Comment:
Optional:
`ids=LANDING_PAGES`
This would give proper id names to all the parametrized tests

Reply:
Not a feasible option here as the name of the page servers as best identifiers and secondly there are around 100 parameters.